website/content/v1.10/advanced/selinux.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "SELinux"
+description: "SELinux security module support (experimental)."
+---
+
+Talos Linux 1.10 introduces initial SELinux support. Security-Enhanced Linux is a security module which
+helps mitigating attacks going past ordinary access control measures like file permission and container namespace isolation.
```

Comment:
I think we need to clarify that what we ship is the base SELinux policies that protects OS/system components from workloads. SELinux itself (as using SELinux in workloads) is not related.